qdrant-landing/content/documentation/infrastructure/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Infrastructure
-weight: 21
-partition: build
+weight: 29
```

Comment:
I don't feel strongly about giving 3rd party tools their own section in the `Cloud` page.